x-pack/filebeat/input/httpjson/encoding.go

```diff
@@ -217,3 +224,13 @@ func decodeAsZip(p []byte, dst *response) error {
 
 	return nil
 }
+
+func decodeAsXML(p []byte, dst *response) error {
+	cdata, v, err := xml.Unmarshal(bytes.NewReader(p), dst.xmlDetails)
```

Comment:
Just a Thought. Can we have more readable variable name instead of `v` just so we don't have to go into `Unmarshall` function to see what it returns

Reply:
I've changed it to body just to show the link to dst.body, but it's not clear to me that it's any clearer.

Reply:
May be `responseBody`

Reply:
This leads to `dst.responseBody`.

Reply:
yeah.. If that sounds OK to you.. Just a `Nit` , so it's upto you